README.md

```diff
@@ -1,7 +1,7 @@
 CBM FLES Timeslice Building
 ===========================
-[![Build Status](https://travis-ci.org/cbm-fles/flesnet.svg?branch=master)](https://travis-ci.org/cbm-fles/flesnet)
-[![Coverage Status](https://coveralls.io/repos/github/cbm-fles/flesnet/badge.svg?branch=master)](https://coveralls.io/github/cbm-fles/flesnet?branch=master)
+[![Build Status](https://travis-ci.org/tschuett/flesnet.svg?branch=fles_libfabric)](https://travis-ci.org/tschuett/flesnet)
```

Comment:
Cannot merge these changes to master without breaking the master's status display.